packages/composites/anchor/test/integration/prices/beth.test.ts

```diff
@@ -80,15 +80,15 @@ jest.mock('ethers', () => {
 
 let oldEnv: NodeJS.ProcessEnv
 
-let server: http.Server
+let server: FastifyInstance
 let req: SuperTest<Test>
 const jobID = '1'
 
 describe('price-beth', () => {
   beforeEach(async () => {
     oldEnv = JSON.parse(JSON.stringify(process.env))
     server = await startServer()
-    req = request(`localhost:${(server.address() as AddressInfo).port}`)
+    req = request(`localhost:${(server.server.address() as AddressInfo).port}`)
```

Comment:
`server.server` reads a bit weird. Does it make sense to rename `server` to `fastify`, `app` or something like that?